### ISSUE TITLE
TypeError - Friendica\Model\Contact::isSharingByURL

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#11798]
- [X ] I have searched open and closed issues for duplicates

### Bug Description
```
Uncaught Exception TypeError: "Argument 1 passed to Friendica\Model\Contact::isSharingByURL() must be of the type string, null given, 
called in .../src/Protocol/ActivityPub/Processor.php on line 920" 
at .../src/Model/Contact.php line 408 
{"exception":"TypeError: Argument 1 passed to Friendica\\Model\\Contact::isSharingByURL() must be of the type string, null given, called in .../src/Protocol/ActivityPub/Processor.php on line 920 and defined in .../src/Model/Contact.php:408
Stack trace:
#0 .../src/Protocol/ActivityPub/Processor.php(920): Friendica\\Model\\Contact::isSharingByURL()
#1 .../src/Protocol/ActivityPub/Receiver.php(634): Friendica\\Protocol\\ActivityPub\\Processor::postItem()
#2 .../src/Protocol/ActivityPub/Queue.php(193): Friendica\\Protocol\\ActivityPub\\Receiver::routeActivities()
#3 .../src/Protocol/ActivityPub/Queue.php(214): Friendica\\Protocol\\ActivityPub\\Queue::process()
#4 .../src/Worker/Cron.php(96): Friendica\\Protocol\\ActivityPub\\Queue::processAll()
#5 [internal function]: Friendica\\Worker\\Cron::execute()
#6 .../src/Core/Worker.php(486): call_user_func_array()
#7 .../src/Core/Worker.php(381): Friendica\\Core\\Worker::execFunction()
#8 .../src/Core/Worker.php(106): Friendica\\Core\\Worker::execute()
#9 .../bin/worker.php(86): Friendica\\Core\\Worker::processQueue()
#10 {main}"} - 
{"file":null,"line":null,"function":null,"uid":"71e02e","process_id":177028}
```

### Platform Info

Friendica Version:
2022.09-dev

Friendica Source: 
git

PHP version:
7.4

SQL version:
MariaDB 10.6

**Comment by MrPetovan** [comment]
Paging @Quix0r

**Comment by Quix0r** [comment]
`$activity['author']` isn't set. I don't know how I can a) fix or b) prevent this (e.g. throwing exception).

**Comment by annando** [comment]
Done in https://github.com/friendica/friendica/pull/11802

**Comment by nupplaphil** [comment]
Closed with #11802
